addons/website_event_track/models/event_track.py

```diff
@@ -603,3 +603,30 @@ def _get_track_suggestions(self, restrict_domain=None, limit=None):
         )
 
         return track_candidates[:limit]
+
+    @api.model
+    def _search_get_detail(self, website, order, options):
+        domain = []
+        if options.get('event'):
+            event_id = self.env['ir.http']._unslug(options['event'])[1]
+            domain.append([('event_id', '=', event_id)])
+
+        fetch_fields = ['name', 'website_url', 'partner_name']
+        mapping = {
+            'name': {'name': 'name', 'type': 'text', 'match': True},
+            'website_url': {'name': 'website_url', 'type': 'text', 'truncate': False},
+            'partner_name': {'name': 'partner_name', 'type': 'text', 'match': True, 'html': True},
+        }
+
```

Comment:
Not needed.

Reply:
hello @ibo-odoo 
I didn't understand this.
Can you please explain :slightly_smiling_face: